truenas/webui
# NAS-125610 / None / Allow full admins to access system settings without meeting 2FA requirements

Testing: 
1. Enable 2FA in global settings as an admin.
2. Observe that you can still access System settings pages, but other normal pages will require you to setup your 2FA device.
